genesis/process/shardGenesisBlockCreator.go

```diff
@@ -292,7 +296,8 @@ func createProcessorsForShardGenesisBlock(arg ArgsGenesisBlockCreator, generalCo
 		DeployEnableEpoch:              arg.GeneralConfig.SCDeployEnableEpoch,
 		AheadOfTimeGasUsageEnableEpoch: arg.GeneralConfig.AheadOfTimeGasUsageEnableEpoch,
 		ArwenV3EnableEpoch:             arg.GeneralConfig.RepairCallbackEnableEpoch,
-		ArwenESDTFunctionsEnableEpoch:  arg.GeneralConfig.ArwenESDTFunctionsEnableEpoch,
+		EpochNotifier:                  epochNotifier,
+		ArwenChangeLocker:              &sync.RWMutex{},
```

Comment:
should be the same pointer with the one on the L394 (on `development` branch I have used the same pointer in these 2 locations)

Reply:
done